### PR TITLE
Fixes #9443: Added `unsigned()` to `ColumnSchemaBuilder`

### LINKED ISSUE #9443: Add support for unsigned integer to migration schema builder
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: yii\db\ColumnSchemaBuilder::unsigned()'

in /var/www/ivm/vendor/yiisoft/yii2/base/Object.php:220

My code:

	public function up() {
		$this->createTable('auditLog', [
			'id' => $this->primaryKey(),
			'model' => $this->string(255)->notNull(),
			'action' => $this->string(255)->notNull(),
			'oldAttributes' => $this->text(),
			'newAttributes' => $this->text(),
			'IPAddress' => $this->string(255),
			'createdAt' => $this->datetime(),
			'updatedAt' => $this->datetime(),
			'createdBy' => $this->integer(10)->unsigned(),
			'updatedBy' => $this->integer(10)->unsigned(),
		]);
	}

### PR DESCRIPTION
**Pull request by samdark**
#9443

### CODE REVIEW

**Comment by samdark** [comment]
@yiisoft/core-developers need review.

File: framework/db/ColumnSchemaBuilder.php [hunk 1]
```diff
@@ -46,6 +46,10 @@ class ColumnSchemaBuilder extends Object
      * @var mixed default value of the column.
      */
     protected $default;
+    /**
+     * @var boolean whether the column values should be unsigned. If this is `true`, a `UNSIGNED` will be added.
```
Comment (SilverFire) [inline framework/db/ColumnSchemaBuilder.php]: a UNSIGNED -> an UNSIGNED

File: framework/db/ColumnSchemaBuilder.php [hunk 2]
```diff
@@ -105,6 +109,16 @@ public function defaultValue($default)
     }
 
     /**
+     * Marks column as unsigned.
+     * @return $this
```
Comment (SilverFire) [inline framework/db/ColumnSchemaBuilder.php]: `@since` ?

**Comment by SilverFire** [comment]
Oracle does not support unsigned integers too

**Comment by samdark** [comment]
Are you sure about Oracle? https://docs.oracle.com/cd/E17952_01/refman-5.5-en/numeric-type-overview.html

**Comment by samdark** [comment]
```
 INT[(M)] [UNSIGNED] [ZEROFILL]

A normal-size integer. The signed range is -2147483648 to 2147483647. The unsigned range is 0 to 4294967295.
```

File: framework/db/ColumnSchemaBuilder.php [hunk 3]
```diff
@@ -203,4 +219,13 @@ protected function buildCheckString()
     {
         return $this->check !== null ? " CHECK ({$this->check})" : '';
     }
+
+    /**
+     * Builds the unsigned string for column.
+     * @return string a string containing UNSIGNED.
```
Comment (SilverFire) [inline framework/db/ColumnSchemaBuilder.php]: Maybe `containing UNSIGNED` -> `containing UNSIGNED keyword`

**Comment by SilverFire** [comment]
Sorry, my fail. Then need to update `yii\db\oci\ColumnSchemaBuilder::_toSting()`

**Comment by samdark** [comment]
Done.

**Comment by cebe** [comment]
looks good in general, would be good to have tests for ColumnSchemaBuilder

**Pull request closed without merging**